apps/explorer/src/components/orders/DetailsTable/index.tsx

```diff
@@ -413,6 +416,18 @@ export function DetailsTable(props: Props): React.ReactNode | null {
               <GasFeeDisplay order={order} />
             </td>
           </tr>
+          <OrderHooksDetails appData={appData} fullAppData={fullAppData ?? undefined}>
+            {(content) => (
+              <tr>
+                <td>
+                  <span>
+                    <HelpTooltip tooltip={tooltip.hooks} /> Hooks
+                  </span>
+                </td>
+                <td>{content}</td>
+              </tr>
+            )}
```

Comment:
Yes, because `content` is a parameter of the function.
It means that  for `OrderHooksDetails` children is a function that has an argument which might be displayed in any place of the children.

```
children: (content: ReactElement) => ReactElement
```

Reply:
Just check the `OrderHooksDetails` internals